docs/data-structures.md

```diff
@@ -1,6 +1,6 @@
 # Data Structures of Cell/Script/Transaction/Block
```

Comment:
Pick the changes to RFC. We'll remove this file.

Reply:
Posted here https://github.com/nervosnetwork/rfcs/pull/91#pullrequestreview-223619365